docs/topics/modules/creating_new_project.adoc

```diff
@@ -48,7 +48,10 @@ image::{context}_application_ready.png[Application ready]
 +
 Your new Vert.X project is now created in your space.
 
-. You have now completed the task, *Launch a sample quickstart application*, in the *Test Iteration*. Ensure that you change the state of the work item to *Closed* using the *Plan* tab.
+Your space dashboard now displays your new application:
+
+image::space_dash_after_hello-world.png[Space dashboard view after creating Hello World]
+
 endif::[]
```

Comment:
@misha, just saw this, not sure why you removed the Step which linked this section to the tasks created in the Planner section?
Remember you suggested making these into steps rather than having them as ending paragraphs.

Reply:
Will re-add it in my next PR